pkg/scheduler/framework/plugins/dynamicresources/dynamicresources_test.go

```diff
@@ -197,6 +198,102 @@ type prepare struct {
 	postfilter change
 }
 
+func BenchmarkHaveAllNodes(b *testing.B) {
+	tests := []*struct {
+		name      string
+		nodeNames []string
+		nodes     []*v1.Node
+		nodeCount int
+	}{
+		{
+			name:      "have-nodes-10",
+			nodeCount: 10,
+		},
+		{
+			name:      "have-nodes-50",
+			nodeCount: 50,
+		},
+		{
+			name:      "have-nodes-100",
+			nodeCount: 100,
+		},
+		{
+			name:      "have-nodes-500",
+			nodeCount: 500,
+		},
+	}
+	for _, test := range tests {
+		for i := 0; i < test.nodeCount; i++ {
+			test.nodeNames = append(test.nodeNames, "worker-"+strconv.Itoa(i))
+			test.nodes = append(test.nodes, &st.MakeNode().Name("worker-"+strconv.Itoa(i)).Node)
+		}
```

Comment:
Add `b.ResetTimer()` here to exclude the initialization time?

Reply:
done.